master/buildbot/status/words.py

```diff
@@ -869,6 +869,10 @@ def command_DANCE(self, args):
         reactor.callLater(3.0, self.send, "(>^.^)>")
         reactor.callLater(3.5, self.send, "(7^.^)7")
         reactor.callLater(5.0, self.send, "(>^.^<)")
+        
+    def command_HUSTLE(self, args):
+        reactor.callLater(1.0, self.send, "/me does the hustle")
+    command_COMMANDS.usage = "dondon on #qutebrowser: qutebrowser-bb needs to learn to do the hustle"  
```

Comment:
It's intended to be rather serious, yes. (as far as this can be considerer serious, anyway)
Sorry for the mistake, I should have been more care full.